tests/playwright/sanity/modules/promotions/helper.ts

```diff
@@ -0,0 +1,9 @@
+import { type Page, expect } from '@playwright/test';
+
```

Comment:
```
readonly page: Page;
	readonly editorPage: EditorPage;
	constructor( page: Page, testInfo: TestInfo ) {
		this.page = page;
		this.editorPage = new EditorPage( this.page, testInfo );
	}
```

Reply:
Can you add a constructor and then use this.page instead of passing it as a params?